helpers/gettempemail.com.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "gettempemail.com",
+  "desc": "create disposable email within seconds",
+  "url": "gettempemail.com",
+  "tags": [
+    "Email",
+    "Misc",
+    "Security"
+  ],
+  "maintainers": [
+    "shivamethicalhat",
+    "gettempemaildev"
```

Comment:
This looks like a generic product account. Could we remove it?

Reply:
sure. both?

Reply:
removed both  :-)